src/apps/content-editor/src/app/views/ItemList/SetRow/ToggleCell/ToggleCell.js

```diff
@@ -26,8 +26,8 @@ export const ToggleCell = memo(function ToggleCell(props) {
               onClick={(evt) => evt.stopPropagation()}
               exclusive
             >
-              <ToggleButton value={0}>False</ToggleButton>
-              <ToggleButton value={1}>True</ToggleButton>
+              <ToggleButton value={0}>{options[0]}</ToggleButton>
```

Comment:
Note: these are not null checked because toggle is inside an if that already checks